brand_external_report_layout/views/report_template.xml

```diff
@@ -0,0 +1,244 @@
+            >o_brand_#{brand_id}_layout o_company_#{company.id}_layout article o_report_layout_striped {{  'o_layout_background' if company.layout_background in ['Geometric', 'Custom']  else  '' }}</attribute>
+        </xpath>
+        <xpath expr="//div[contains(@t-attf-class, 'footer')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >o_brand_#{brand_id}_layout o_company_#{company.id}_layout footer o_background_footer</attribute>
+        </xpath>
+    </template>
+    <template id="external_layout_bold" inherit_id="web.external_layout_bold">
+        <xpath expr="//div[contains(@t-attf-class, 'header')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >header o_brand_#{brand_id}_layout o_company_#{company.id}_layout</attribute>
+        </xpath>
+        <xpath expr="//div[contains(@t-attf-class, 'article')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >article o_report_layout_bold o_brand_#{brand_id}_layout o_company_#{company.id}_layout {{  'o_layout_background' if company.layout_background in ['Geometric', 'Custom']  else  '' }}']</attribute>
+        </xpath>
+        <xpath expr="//div[contains(@t-attf-class, 'footer')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >footer o_bold_footer o_brand_#{brand_id}_layout o_company_#{company.id}_layout</attribute>
+        </xpath>
+    </template>
+    <template id="external_layout_standard" inherit_id="web.external_layout_standard">
+        <xpath expr="//div[contains(@t-attf-class, 'header')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >header o_brand_#{brand_id}_layout o_company_#{company.id}_layout</attribute>
+        </xpath>
+        <xpath expr="//div[contains(@t-attf-class, 'article')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >article o_report_layout_standard o_brand_#{brand_id}_layout o_company_#{company.id}_layout {{  'o_layout_background' if company.layout_background in ['Geometric', 'Custom']  else  '' }}']</attribute>
+        </xpath>
+        <xpath expr="//div[contains(@t-attf-class, 'footer')]" position="attributes">
+            <attribute
+                name="t-attf-class"
+            >footer o_standard_footer o_brand_#{brand_id}_layout o_company_#{company.id}_layout</attribute>
+        </xpath>
+    </template>
+
+    <record id="asset_styles_brand_report" model="ir.attachment">
+        <field name="datas" model="res.brand" eval="obj()._get_asset_style_b64()" />
+        <field name="mimetype">text/scss</field>
+        <field name="name">res.brand.scss</field>
+        <field name="type">binary</field>
+        <field
+            name="url"
+        >/brand_external_report_layout/static/src/legacy/scss/asset_styles_brand_report.scss</field>
```

Comment:
Same here. 
This file is no longer there..
Seems to be introduced in https://github.com/OCA/brand/pull/109